yaml/pois.yaml

```diff
@@ -2640,6 +2640,11 @@ filters:
                 - shiploader
                 - tower_crane
             then: {col: "crane:type"}
+  - filter: {man_made: chimney}
+    min_zoom: 14
```

Comment:
Please vary the min_zoom, like with like for obelisk:

```
  - filter: {man_made: obelisk}
    min_zoom:
      lookup:
        key: { call: { func: mz_to_float_meters, args: [ { col: height } ] } }
        op: '>='
        table:
          - [ 14, 20 ]  # z14 if height >= 20m
          - [ 15, 10 ]  # z15 if height >= 10m
        default: 16
```

Reply:
Ok, I have changed it as is in Obelisk.